frontend/lib/src/hostComm/HostCommunicationManager.tsx

```diff
@@ -14,7 +14,7 @@
  * limitations under the License.
  */
 
-import { ICustomThemeConfig, WidgetStates } from "src/proto"
+import { ICustomThemeConfig, WidgetStates } from "../proto"
```

Comment:
nit: Similar to the comment above, what is the general rule: Are we fine with relative path if it is one level back?

Reply:
Maybe we only allow relative paths if it is within the same directory or child directory